mpas_analysis/ocean/meridional_overturning_circulation.py

```diff
@@ -400,65 +426,68 @@ def _compute_moc_postprocess(config, streams, variableMap, calendar,
-                                      regionCellMask, transportZ, velArea)
-                mocAtlantic26[ncount] = np.amax(mocTop[:, indlat26])
-                ncount += 1
-
-            Time[ncount-12:ncount] = ds.Time[ncount-12:ncount].values
-
-            # Save to file
-            print '   save to yearly file...'
-            ncFile = netCDF4.Dataset(outputFileTseries, mode='w')
-            # create dimension
-            ncFile.createDimension('Time', 0)
-            # create variables
-            Timevar = ncFile.createVariable('Time', 'f4', ('Time',))
-            Timevar.description = 'days since start of simulation'
-            mocAtlantic26var = ncFile.createVariable('mocRapidArrayMax',
-                                                     'f4', ('Time',))
-            mocAtlantic26var.description = 'Max MOC Atlantic streamfunction'\
-                                           ' nearest to RAPID Array latitude'\
-                                           ' (26.5N)'
-            mocAtlantic26var.units = 'Sv (10^6 m^3/s)'
-            # save variables
-            Timevar[:] = Time[ncount-12:ncount]
-            mocAtlantic26var[:] = mocAtlantic26[ncount-12:ncount]
-            ncFile.close()
-        else:
-            ncount += 12
-            # Read from file
-            print '     Read previously computed Atlantic MOC at 26.5N'\
-                  ' from file...'
-            ncFile = netCDF4.Dataset(outputFileTseries, mode='r')
-            Time[ncount-12:ncount] = ncFile.variables['Time'][:]
-            mocAtlantic26[ncount-12:ncount] = ncFile.variables['mocRapidArrayMax'][:]
-            ncFile.close()
-
-    # Store max Atlantic MOC at 26.5 to dictionary
-    mocDictTseries = {'Time': {'dims': ('Time'), 'data': Time},
-                      'mocRapidArrayMax': {'dims': ('Time'),
-                                           'data': mocAtlantic26}}
-
-    return (mocDictClimo, mocDictTseries)
+    outputFileTseries = '{}/mocTimeSeries.nc'.format(outputDirectory)
+
+    continueOutput = os.path.exists(outputFileTseries)
+    if continueOutput:
+        # read in what we have so far
+        dsMOCTimeSeries = xr.open_dataset(outputFileTseries)
+        # force loading and then close so we can overwrite the file later
+        dsMOCTimeSeries.load()
+        dsMOCTimeSeries.close()
+    else:
+        # create a new data set from a dictionary
+        dictonary = {
+            'Time': {'dims': ('Time'),
+                     'units': 'days since 0001-01-01',
+                     'data': []},
+            'mocAtlantic26': {'dims': ('Time'),
+                              'units': 'Sv (10^6 m^3/s)',
+                              'description': 'Max MOC Atlantic streamfunction '
+                                             'nearest to RAPID Array '
+                                             'latitude (26.5N)',
+                              'data': []}}
```

Comment:
@xylar: it would be nice to add this information to the variable attributes, as it was before. Did you figure out how to do it with xarray? I couldn't.